### DISCUSSION TITLE
Is there a way to make particles return to orginal position?

### DISCUSSION THREAD

**Discussion by aidenBarrett96** [matteobruni/tsparticles#1233] [Q&A]
I would like to have a polygon mask of text that will expand and do its particle thing, then when hovered on the original path the particles will return to spell out the text

**Comment by matteobruni** [comment]
Hi @aidenBarrett96,

I already thought about this possibility but I'm still thinking about how it can be easily implemented and used. Probably a plugin will come out like the `polygon mask`.
I'm sorry but this feature it's not a priority at the moment since I'd like to release the new `2.0` version but it still needs a lot of work since I'm working on the composability.

**Reply by aidenBarrett96** [reply]
That's no problem, thanks for the response anyway
